gammapy/image/utils.py

```diff
@@ -387,7 +387,7 @@ def coordinates(image, world=True, lon_sym=True, radians=False):
         y, x = np.indices(image.data.shape, dtype='int32') + 1
 
     wcs = WCS(image.header)
-    lon, lat = wcs.wcs_pix2world(x, y, 1)
+    lon, lat = wcs.wcs_pix2world(x, y, 0)
```

Comment:
Ok, you can then take over. If you modify the `coordinates` function in `gammapy/image/utils.py` in the way you mention, you may want to revert the changes in the hard coded references to the number of pixels in `test_image_lon_profile` from ` gammapy/image/tests/test_profile.py `.